doc/classes/Array.xml

```diff
@@ -510,14 +510,14 @@
 			<argument index="1" name="accum" type="Variant" default="null">
 			</argument>
 			<description>
-				Calls the provided [Callable] for each element in array and accumulates the result in [code]accum[/code]. The method for [Callable] takse two arguments: current value of [code]accum[/code] and the current array element. If [code]accum[/code] is [code]null[/code] (default value), the method will use first element from the array as initial value.
+				Calls the provided [Callable] for each element in array and accumulates the result in [code]accum[/code]. The method for [Callable] takes two arguments: current value of [code]accum[/code] and the current array element. If [code]accum[/code] is [code]null[/code] (default value), the method will use the first element from the array as initial value.
 				[codeblock]
 				func _ready():
-				    print([1, 2, 3].reduce(factorial, 1)) # Prints 6.
-				    print([1, 2, 3].reduce(func(accum, number): return accum * number)) # Same as above, but using lambda function.
+				    print([1, 2, 3].reduce(sum, 1)) # Prints 6.
```

Comment:
Shouldn't it print 7? Since you pass `1` as `accum`, which then gets summed with `1 + 2 + 3`?

Might be worth using different numbers which don't create ambiguity.

E.g. `print([1, 2, 3].reduce(sum, 10)) # Prints 16.`